### ISSUE TITLE
Generated code formatting causes error with field names containing space

### ISSUE THREAD

**Issue by Ezard** [square/moshi#1117]
Currently, there seems to be some auto-formatting of generated code, with respect to maximum line length

However, if strings this can cause strings to be wrapped, which causes compilation errors

**Example:**
```
@JsonClass(generateAdapter = true)
class Example(
  @Json(name = "Some Thing") val a: String,
  @Json(name = "Some Thing Else") val b: String,
  @Json(name = "Foo Bar") val c: String
)
```
generates
```
private val options: JsonReader.Options = JsonReader.Options.of("Some Thing", "Some Thing
    Else", "Foo Bar")
```

**Solution:**
Avoid wrapping this line

**Comment by rharter** [comment]
This looks like it's probably a KotlinPoet issue.

**Comment by rharter** [comment]
I've got a test case in the works for this, but can't reproduce it.  Even going to extremes, I can't get the code generator to break within a string.  Here's what I see generated:

```kotlin
  private val options: JsonReader.Options =
      JsonReader.Options.of("a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a",
      "b b b b b b b b b b b b b b b b b b b b b b b b b b b b b b b b",
      "Some Other Else or some other thing")
```

Would you mind seeing if you can reproduce this in the current master branch, and providing a reproducible case, if so?

**Comment by Ezard** [comment]
@rharter sorry for the slow response - I'll see if I can put something together

**Comment by antonkovalyov351** [comment]
Got same issue:

Test class:

```
import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class TestJsonClass(
    @Json(name = "Long Field Name With Spaces 1") val field1: Int,
    @Json(name = "Long Field Name With Spaces 2") val field2: Int,
    @Json(name = "Long Field Name With Spaces 3") val field3: Int,
    @Json(name = "Long Field Name With Spaces 4") val field4: Int,
    @Json(name = "Long Field Name With Spaces 5") val field5: Int
)
```

**Comment by antonkovalyov351** [comment]
Any news?

**Comment by ZacSweers** [comment]
This has been fixed since 1.9.3 via #1053
